src/components/event/tables/ParamsTable.tsx

```diff
@@ -81,6 +81,26 @@ class ParamsTableBase extends React.Component<Props, State> {
 		};
 	}
 
+	setExpandStatus(isCollapsed: boolean) {
+		this.setState({
+			nodes: this.state.nodes.map(node =>
+				node.subRows ? this.setNodeExpandStatus(node, isCollapsed) : node,
```

Comment:
node.subRows always be truthy and second case will never be done. You can check node.subRows.length or just execute setNodeExpandStatus